syntax = "proto3";

package train;

service TrainingService {
  rpc TrainModel (TrainRequest) returns (TrainResponse);
}

message TrainRequest {
  repeated double x_train = 1;
  repeated double y_train = 2;
}

message TrainResponse {
  bytes model = 1;
}
